src/pages/Home.js

```diff
@@ -0,0 +1,38 @@
+import {useState} from 'react';
+import '../App.css';
+import Header from '../components/Header/Header.js';
+import {Search} from '../components/Search/Search';
+// import Navigation from './components/Navigation/Navigation';
+import SearchResults from '../components/SearchResults/SearchResults';
+import productsDatabase from '../database.js';
+
+export default function Home() {
+  const [searchTerm, setSearchTerm] = useState('');
+  function handleSearchTermChange(changedTerm) {
+    setSearchTerm(changedTerm);
+  }
+
+  const filteredProducts = productsDatabase.reduce((prev, cur) => {
```

Comment:
try not to use short words. I guess this is previous and current. 
Its always easier to read when you use descriptive words.

Reply:
@julialoeschel Coach Ralf wrote this.